app/controllers/api/v1/jets_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Api::V1::JetsController < ApplicationController
+  def index
```

Comment:
- Why do the controllers not use `before_action` command? Without this, everyone (logged or not) is able